src/Data/Auth/AuthProvider.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { AuthConfig, KeycloakProvider, LocalConfig } from "./KeycloakProvider";
+import { DatabaseAuthProvider } from "./PartialProviders/DatabaseAuthProvider";
```

Comment:
PartialProviders? Providers was just fine as folder name. You just need to put the right files under the right name. I'm not even sure what I need to understand under PartialProviders.